tests/ImageSharp.Tests/Drawing/DrawPathTests.cs

```diff
@@ -32,13 +33,13 @@ public void ImageShouldBeOverlayedByPath()
                     new Vector2(60, 10),
                     new Vector2(10, 400));
 
-                CorePath p = new CorePath(linerSegemnt, bazierSegment);
+            	ShapePath p = new ShapePath(linerSegemnt, bazierSegment);
```

Comment:
Why are we creating new shape paths for the unit tests? Are we expecting developers to have to do the same in their own code?